lib/services/versions-service.ts

```diff
@@ -1,22 +1,28 @@
-import { EOL } from "os";
 import * as constants from "../constants";
+import * as helpers from "../common/helpers";
 import * as semver from "semver";
 import * as path from "path";
-import { createTable } from "../common/helpers";
+
+export enum VersionInformationType {
+	UpToDate,
```

Comment:
You could assign values here, should you decide to follow my suggestion in `mobile-cli-lib`'s PR